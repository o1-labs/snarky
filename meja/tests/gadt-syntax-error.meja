type t = A('a) : t;
