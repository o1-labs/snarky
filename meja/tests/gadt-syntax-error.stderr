File "tests/gadt-syntax-error.meja", line 1, characters 9-18:
Error: GADT case syntax cannot be used in a non-recursive type.
       To use this syntax, use 'type rec' for this type definition.

